app/src/main/resources/produsent.graphql

```diff
@@ -195,12 +195,124 @@ enum EksterntVarselStatus {
     FEILET
 }
 
+input NySakInput {
+    """
+    Grupperings-id-en knytter en sak og notifikasjoner sammen.
+    Den skal være for unik for saker innenfor merkelappen.
+    Et naturlig valg av grupperingsid er f.eks. et saksnummer.
+    """
+    grupperingsid: String!
+
+    """Merkelapp som saken skal assossieres med."""
+    merkelapp: String!
+
+    """Virksomhetsnummeret til virksomheten som saken omhandler."""
+    virksomhetsnummer: String!
+
+    """
+    Hvem som skal få se saken.
+
+    NB. At en bruker har tilgang til en sak påvirker ikke om de har tilgang
+    til en notifikasjon. De tilgangsstyres hver for seg.
+    """
+    mottakere: [MottakerInput!]!
+
+    """En tittel på saken, som vises til brukeren."""
+    tittel: String!
+
+    """Her oppgir dere en lenke som brukeren kan klikke på for å komme rett til saken."""
+    lenke: String!
+
+    """Status saken starter med."""
+    status: StatusOppdateringInput!
+}
+
+union NySakResultat =
+    | NySakVellykket
+    | UgyldigMerkelapp
+    | UgyldigMottaker
+    | DuplikatGrupperingsid
+    | UkjentProdusent
+
+type NySakVellykket {
+    id: ID!
+}
+
+input OppdaterSakInput {
+    status: StatusOppdateringInput!
+}
+
+union OppdaterSakResultat =
+    | OppdaterSakVellykket
+    | SakFinnesIkke
+
+type OppdaterSakVellykket {
+    id: ID!
+
+    """Nyeste statusoppdatering er først i listen."""
+    statuser: [StatusOppdatering!]!
+}
+
+"""
+Statusen påvirker bl.a. hvilket ikon som vises og brukes bl.a.
+for å kunne filtrere saksoversikten på min side arbeidsgiver.
+"""
+enum SaksStatus {
+    """
+    Naturlig start-tilstand for en sak.
+
+    Default tekst som vises til bruker: "Mottatt"
+    """
+    MOTTATT
+
+    """
+    Default tekst som vises til bruker: "Under behandling"
+    """
+    UNDER_BEHANDLING
+
+    """
+    Slutt-tilstand for en sak. Når en sak er `FERDIG`, så vil vi
+    nedprioritere visningen av den på min side arbeidsgivere.
+
+    Default tekst som vises til bruker: "Ferdig".
+    """
+    FERDIG
+}
+
+input StatusOppdateringInput {
+    status: SaksStatus!
+
+    """
+    Når endringen skjedde.
+    Dette feltet er frivillig. Hvis feltet ikke er oppgitt, bruker vi tidspunktet dere gjør
+    kallet på.
```

Comment:
skal vi nevne at det er lov å sette tidspunkt i fortiden?

Reply:
Ja, det kan vi. Men hva med fremtiden? Det er kanskje ikke lov? Men er kanskje noe vi aksepterer i det stille? Vil tro det blir naturlig å vise statusoppdateringer sortert på tid, ikke når vi mottok dem (selv om jeg regner med det sammenfaller som oftest)

Reply:
ja, kanskje bare akseptere det, mulig det blir aktuelt hvis vi skal implementere planlagte statusendringer, men det er kanskje et eget konsept.

Reply:
Tenker umiddelbart at man kanskje har lyst til å skille tydelig mellom ting som "har skjedd" og ting som "forventes å skje" i api-et. Ting som "har skjedd" er vel mer immutable, mens for det som forventes, så vil man kanskje overskrive.